pyramid_assemble.py

```diff
@@ -236,8 +239,17 @@ def main():
 
         print()
 
+    if dtype == np.uint32 and is_mask:
+        ome_dtype = 'uint32'
+    elif dtype == np.uint16:
+        ome_dtype = 'uint16'
+    elif dtype == np.uint8:
+        ome_dtype = 'uint8'
+    else:
+        raise ValueError("can't handle dtype: %s" % dtype)
```

Comment:
This should become a print and `sys.exit(1)`.

Reply:
Fixed